api-interop-layer/util/fetch.js

```diff
@@ -0,0 +1,53 @@
+import { createLogger } from "./monitoring/index.js";
+import { sleep } from "./sleep.js";
+
+const logger = createLogger("fetch wrapper");
+
+const BASE_URL = process.env.API_URL ?? "https://api.weather.gov";
+
+const internalFetch = async (path) => {
+  const url = URL.canParse(path) ? path : new URL(path, BASE_URL).toString();
+  logger.verbose(`making request to ${url}`);
+
+  return fetch(url).then(async (r) => {
```

Comment:
Still thinking about this. It might be a legit concern since the first outbound call we make is dependent on the lat/lon values passed as URL path components. All subsequent calls are based on the response to that one. I don't think path-walking should be a concern

Reply:
Oh, no, never mind, fastify is enforcing constraints on those values for us.